packages/ui/src/composites/wui-dropdown-menu/styles.ts

```diff
@@ -0,0 +1,18 @@
+import { css } from 'lit'
+
+export default css`
+  :host {
+    z-index: calc(var(--w3m-z-index) + 1);
+    width: 200px;
+    padding: var(--wui-spacing-3xs);
+    align-items: center;
+    display: inherit;
+    border-radius: var(--wui-border-radius-xs);
+    border: 1px solid var(--wui-gray-glass-002);
+    background: var(--wui-color-fg-base-125);
+    /* Dark/Elevation/L */
```

Comment:
Do we need to create CSS variable for this shadow value? Is this `Dark/Elevation/L` coming from the design system? Could you share a link for the designs?

Reply:
@tomiir will be best to answer

Reply:
Doesn't seem to be a standardized shadow in the ThemeUtil. We use the same shadow on the snackbar so we could maybe abstract it but don't think it's a big deal. If we do I'd do it as part of design system variable review in general which we could do on tech debt week imo